appveyor.yml

```diff
@@ -11,21 +11,21 @@ branches:
 install:
   - ps: |
       Add-Type -AssemblyName System.IO.Compression.FileSystem
-      if (!(Test-Path -Path "C:\maven\apache-maven-3.2.5" )) {
+      if (!(Test-Path -Path "C:\maven\apache-maven-3.8.1" )) {
```

Comment:
if it is possible, please extract version to some constant and replace usages with constant

Reply:
yml files don't support placeholder unline dockerfile and bash. So, I don't think it is possible.

Reply:
I don't think it is possible after reading docs.